docs/UserGuide.md

```diff
@@ -38,14 +38,15 @@ Expense Expert is a **desktop app for keeping track of your expenses, and managi
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `ExpenseExpert.jar` from [here](https://github.com/AY2122S2-CS2103T-W09-3/tp/releases).
+2. Download the latest `expenseexpert.jar` from [here](https://github.com/AY2122S2-CS2103T-W09-3/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your Expense Expert.
+3. Copy the file to the folder you want to use as the _home folder_ for your Expense Expert.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. 
+5. The data is just for show, in reality the application will be blank in the beginning.<br>
```

Comment:
This sounds really bad. I think "data is for demonstration purpose" and I think the data won't be blank at the start, theres SampleDataUtil.

Reply:
changed the wording

Reply:
Okay ..., didn't realised that.